android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java

```diff
@@ -597,6 +725,37 @@ public void reportEndCallWithUUID(String uuid, int reason) {
         conn.reportDisconnect(reason);
     }
 
+   @Override
+   public void onHostResume() {
+       Log.d(TAG, "onResume()");
+   }
+
+   @Override
+   public void onHostPause() {
+       Log.d(TAG, "onPause()");
+   }
```

Comment:
Can we remove this ?

Reply:
Unforitunely no, if i removed them the build will failed, if you overrided one of them you to override the three of them

Reply:
Ok so could you remove the log please ?